.changelog/23259.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+resources: Added new oom_score_adj field
+```
```

Comment:
We haven't wired this up to the drivers yet, so if we merge this and end up shipping Nomad 1.8.1 before that work is complete, the changelog entry won't make much sense. Maybe we should hold off adding the changelog entry until all the components (and documentation!) are in place? That way we can safely ship this code in `main` without stressing about 1.8.1.

Reply:
i can also remove the backport labels for now.